pages/checkIn.tsx

```diff
@@ -1,5 +1,60 @@
+import CheckInOutForm from 'components/CheckInOutForm'
+import {
+  Box,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Typography,
+  Unstable_Grid2 as Grid2,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+
 export default function CheckInPage() {
+  const theme = useTheme()
+  const isMobileView = useMediaQuery(theme.breakpoints.down('sm'))
   return (
-    <h1>Check In</h1>
+    <Grid2 container sx={{ flexGrow: 1 }}>
+      <Grid2
+        xs={12}
+        sm={8}
+        lg={6}
+        display="flex"
+        justifyContent="flex-end"
+        smOffset={2}
+        lgOffset={3}
+      >
+        <Button
+          variant="outlined"
+          fullWidth={isMobileView}
+          size="large"
+          sx={{ my: 2 }}
+        >
+          Create new item
+        </Button>
+      </Grid2>
+
+      <Grid2 xs={12} sm={8} lg={6} smOffset={2} lgOffset={3}>
+        <Card variant={isMobileView ? 'elevation' : 'outlined'} elevation={0}>
+          <Box display="flex" flexDirection="column">
+            <CardContent sx={{ p: isMobileView ? 0 : 2 }}>
+              <Typography variant="h5" sx={{ mb: 2 }}>
+                Check in items
+              </Typography>
+              <CheckInOutForm
+                kioskMode={true}
+                users={[]}
+                itemDefinitions={[]}
+              />
+            </CardContent>
+
+            <CardActions sx={{ alignSelf: { xs: 'end' } }}>
```

Comment:
Add `mt: { xs: 1, sm: 0 }` so that there's a bit more space between the quantity component and the checkin button on mobile, and this will be good to go!

![image](https://user-images.githubusercontent.com/71574118/221329598-bc72610c-a679-4344-875b-598dc0630989.png)